src/font.c

```diff
@@ -195,8 +199,12 @@ GdipDeleteFontFamily (GpFontFamily *fontFamily)
 	if (!fontFamily)
 		return InvalidParameter;
 
+#if GLIB_CHECK_VERSION(2,32,0)
```

Comment:
I've seen this pattern all across the file. Maybe it's worth creating our own macro/function for this to avoid all these checks?

Reply:
yeah, makes sense.